src/features/projectsV2/ProjectDetails/components/OtherInterventionInfo.tsx

```
@@ -0,0 +1,183 @@
+const OtherInterventionInfo = ({
+  plantLocationInfo,
+  isMobile,
+  setSelectedSamplePlantLocation,
+}: Props) => {
+  const tProjectDetails = useTranslations('ProjectDetails');
+
+
+  const { totalTreesCount } = useMemo(() => {
+    const totalTreesCount =
+      plantLocationInfo &&
+        plantLocationInfo.plantedSpecies &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)